src/components/views/settings/account/EmailAddresses.js

```diff
@@ -193,8 +193,14 @@ export default class EmailAddresses extends React.Component {
             this.props.onEmailsChange(emails);
         }).catch((err) => {
             this.setState({continueDisabled: false});
-            if (err.errcode !== 'M_THREEPID_AUTH_FAILED') {
-                const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
+            const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
+            if (err.errcode === 'M_THREEPID_AUTH_FAILED') {
+                Modal.createTrackedDialog("E-mail hasn't been verified yet", "", ErrorDialog, {
```

Comment:
In our strings, we consistently use "email" instead of "e-mail", so please change all these to "email".